uwblueprint/jump-math
# fix: refresh token issues

## Notion ticket link

[Fix refresh token](https://www.notion.so/uwblueprintexecs/Fix-refresh-token-d6c6eccfa072419a8282889910804221)



## Implementation description
- Remove unused component
- Chain refresh -> logout -> clear storage
- Don't log out upon a network error



## Steps to test
Testing that a missing refresh token is handled:
1. Log in to your local environment.
2. Set your system time to a time at least a week in the future.
3. Close your browser fully and reopen it.
4. Load any page that pulls data from the backend, and verify that you are logged out.

Testing that refresh tokens still work:
1. Log in to your local environment.
2. Set your system time to a time one day in the future.
3. Open the network tab.
3. Navigate to any page that pulls data from the backend, and verify that (a) the data still shows up (i.e., you aren't logged out), and (b) a network request was made for a new access token.



## What should reviewers focus on?
* 


## Checklist
